ajax/libs/popper.js/package.json

```
@@ -1,11 +1,11 @@
 {
   "name": "popper.js",
   "filename": "popper.min.js",
-  "version": "1.16.1",
+  "version": "2.3.3",
```

```suggestion
  "version": "1.16.1",
```
Please revert this change, the auto-update bot handles changing the version when it pulls inn new versioned files.